tidb-cloud/migration-job.md

```diff
@@ -0,0 +1,185 @@
+
+### Set up network connection
+
+Before creating a migration job, set up the network connection according to different connection types. See [Connect to Your TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster).
+
+- If you use public IP for network connection, make sure that the upstream and downstream databases can be connected through the public network.
+
+- If you use VPC Peering, set it up according to [Add VPC peering requests](/tidb-cloud/set-up-vpc-peering-connections.md#step-1-add-vpc-peering-requests).
+
+If you use AWS PrivateLink, set it up according to [Set Up Private Endpoint Connections](/tidb-cloud/set-up-private-endpoint-connections.md).
+
+### Enable binlog
+
+If you perform incremental data migration, make sure you have enabled binlog, and the binlogs have been kept for more than 24 hours.
+
+## Step 1: Go to the **Data Migration** page
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com/console/clusters). Navigate to the **Clusters** page for your project.
+
+2. On the **Clusters** page, click the name of your cluster, and click the **Data Migration** tab. The **Data Migration** page is displayed.
+
+3. On the **Data Migration** page, click **Create Migration Job**. The **Create Migration Job** page is displayed.
+
+## Step 2: Configure the source and target connection
+
+On the **Create Migration Job** page, configure the source and target connection.
+
+1. Enter a job name, which must start with a letter and must be less than 60 characters. Letters (A-Z, a-z), numbers (0-9), underscores (_), and hyphens (-) are acceptable.
+
+2. Fill in the source connection profile.
+
+   - **Data source**: the data source type.
+   - **Region**: the region of the data source, which is required for cloud databases only.
+   - **Connectivity method**: the connectivity method for the data source. Currently, public IP, VPC Peering, or Private Private Link is supported.
+   - **Hostname or IP address** (for public IP and VPC Peering): the hostname or IP address of the data source.
+   - **Service Name** (for Private Link): the endpoint service name.
+   - **Port**: the port of the data source.
+   - **Username**: the username of the data source.
+   - **Password**: the password of the username.
+   - **SSL/TLS**: whether to use SSL/TLS connection for the data source. If you enable SSL/TLS, you need to upload the certificates of the data source, including the CA certificate, client certificate, and client key.
+
+3. Fill in the target connection profile.
+
+   - **Username**: enter the username of the target cluster in TiDB Cloud.
+   - **Password**: enter the password of the TiDB Cloud username.
+
+4. Click **Validate Connection and Next** to validate the information you have entered.
+
+5. Take action according to the message you see:
+
+    - If you use Public IP or  VPC Peering, you need to add the Data Migration service's IP addresses to the IP Access List of your source database and firewall (if any).
```

Comment:
```suggestion
    - If you use Public IP or VPC Peering, you need to add the Data Migration service's IP addresses to the IP Access List of your source database and firewall (if any).
```

Reply:
Question: Is this feature available for Serverless Tier? Serverless Tier does not support IP Access List.

Reply:
No. Only for dedicated tier.